### ISSUE TITLE
Cast causes build to fail

### ISSUE THREAD

**Issue by rcurtin** [viva64/how-to-use-pvs-studio-free#10]
Hi there,

We use this project to run static code checks on our build.  It works great, but our builds have started to fail because of the most recent commit:

```
[ 75%] Building CXX object CMakeFiles/how-to-use-pvs-studio-free.dir/encoding.cpp.o
In file included from /home/jenkins/jobs/pull-requests-mlpack-static-code-analysis/workspace/pvs-util/comments.cpp:4:0:
/home/jenkins/jobs/pull-requests-mlpack-static-code-analysis/workspace/pvs-util/comments.h: In member function ‘bool PvsStudioFreeComments::Comment::addLine(const char*, const char*)’:
/home/jenkins/jobs/pull-requests-mlpack-static-code-analysis/workspace/pvs-util/comments.h:31:55: error: use of old-style cast [-Werror=old-style-cast]
       while (begin != end && isspace(*(unsigned char*)begin))
                                                       ^
/home/jenkins/jobs/pull-requests-mlpack-static-code-analysis/workspace/pvs-util/comments.h:42:63: error: use of old-style cast [-Werror=old-style-cast]
       while (begin != end && isspace(*(unsigned char*)(end - 1)))
                                                               ^
In file included from /home/jenkins/jobs/pull-requests-mlpack-static-code-analysis/workspace/pvs-util/main.cpp:4:0:
/home/jenkins/jobs/pull-requests-mlpack-static-code-analysis/workspace/pvs-util/comments.h: In member function ‘bool PvsStudioFreeComments::Comment::addLine(const char*, const char*)’:
/home/jenkins/jobs/pull-requests-mlpack-static-code-analysis/workspace/pvs-util/comments.h:31:55: error: use of old-style cast [-Werror=old-style-cast]
       while (begin != end && isspace(*(unsigned char*)begin))
                                                       ^
/home/jenkins/jobs/pull-requests-mlpack-static-code-analysis/workspace/pvs-util/comments.h:42:63: error: use of old-style cast [-Werror=old-style-cast]
       while (begin != end && isspace(*(unsigned char*)(end - 1)))
                                                               ^
cc1plus: all warnings being treated as errors
CMakeFiles/how-to-use-pvs-studio-free.dir/build.make:86: recipe for target 'CMakeFiles/how-to-use-pvs-studio-free.dir/comments.cpp.o' failed
```

Here is the full output:
http://masterblaster.mlpack.org/job/pull-requests-mlpack-static-code-analysis/715/console

We can work around it if we need, but I figured it was better to report it. :)

Thanks!

**Comment by SvyatoslavRazmyslov** [comment]
Fix build bfd342154361513baa2d02e6fc6afde58d101e8a.

Thanks!

**Comment by rcurtin** [comment]
Thanks so much for the quick fix! :)
